selfdrive/debug/print_docs_diff.py

```diff
@@ -107,11 +109,15 @@ def print_car_info_diff(path):
           markdown_builder.append(COLUMN_HEADER)
         markdown_builder.extend(changes[category])
 
-    print("\n".join(markdown_builder))
+    diff = "\n".join(markdown_builder)
+    if github:
+      diff = diff[:MAX_GITHUB_COMMENT_SIZE] + '\n...\nTo see the full diff, run ``` python selfdrive/debug/print_docs_diff.py ```'
```

Comment:
does `MAX_GITHUB_COMMENT_SIZE` take into account the extra message added?

Reply:
yes it does